### ISSUE TITLE
A question about the Arbiter deployment.

### ISSUE THREAD

**Issue by gxcuit** [FederatedAI/FATE#3621]
In the [deployment doc](https://github.com/FederatedAI/FATE/blob/master/deploy/cluster-deploy/doc/fate_on_eggroll/fate-allinone_deployment_guide.md), there is only config for the Guest and the Host. What about the Arbiter? In the real application, **where is the Arbiter should be deployed?** It is seems that the Arbiter   has the secret key, and can obtain the plaintext of the host's and the guest's model in the Hetero settings.  I noticed in the [hetero-lr examples](https://github.com/FederatedAI/FATE/blob/master/examples/dsl/v2/hetero_logistic_regression/hetero_lr_normal_conf.json), the Arbiter has the same id with the Host.  **Does this mean the Arbiter is deployed on the same machine with the Host**? If so, can the malicious Host get the Arbiter's result? 

**In summary**, my questions is : If we do not deploy the Arbiter in an independent machine, can the malicious party get the message that the Arbiter received/computed, or the secret key that the arbiter generated? 

Looking for the reply. Thanks!

**Comment by gxcuit** [comment]
@dylan-fan
